app/static/error.html

```diff
@@ -21,16 +24,20 @@ <h1>GitHub Desktop failed to launch &#x1F4A5;</h1>
       </p>
 
       <p>
-        Unfortunately this isn't something that can be forwarded to the development
-        team to triage.
+        Unfortunately this isn't something that can be forwarded to the
+        development team to triage.
       </p>
 
       <p>
-        Would you mind opening an issue on the GitHub Desktop <a onclick='return onClick(event);' href='https://github.com/desktop/desktop/issues'>issue tracker</a>?
-        The details you need are included below:
+        Would you mind opening an issue on the GitHub Desktop
+        <a
+          onclick="return onClick(event);"
+          href="https://github.com/desktop/desktop/issues"
+          >issue tracker</a
+        >? The details you need are included below:
       </p>
 
-      <!--{{content}}-->
+      <!-- {{content}} -->
```

Comment:
I need to remember where/how we're setting this placeholder and confirm the introduced whitespace isn't going to break it

Reply:
Is this file even used anymore? It looks like the `crash` webpack config generates its own HTML.

Reply:
I also don’t recall seeing this wording when Desktop crashed in the past.

Reply:
There's one way to find out 🤔

Reply:
Aha, I knew this was something I was involved with: https://github.com/desktop/desktop/pull/1198#issuecomment-295160162

Reply:
The component that rendered this was removed in c76110b34dec2adcd9d375114cfa802f1d312c01 so I'm going to remove it from this PR to avoid any further confusion.